modules/Authentication.php

```diff
@@ -49,18 +77,12 @@ public static function withEmailPassword($db, $email, $password)
         return null;
     }
 
-    public function getUserIdFromSessionToken($session_token)
-    {
-        $stmt = $this->db->prepare("SELECT * FROM `user_session` WHERE `session_token` = ?");
-        $stmt->bindParams('i', $session_token);
-        $stmt->execute();
-        $result = $stmt->getResults();
-        $row = $result->fetchAssoc();
-        $stmt->close();
-        return $row;
-    }
-
-    // Create session token and adding log in database for validation
+    /**
+     * @method : createSession
```

Comment:
The example I posted was give an idea about the fields that need to be in the comment, not the '@' format. The block comment needs to capture details of the method.